src/node-lib/curriculum-api-2023/queries/curriculumSubjectPhaseOverview/overview.query.ts

```diff
@@ -3,7 +3,16 @@ import { Sdk } from "../../sdk";
 
 const curriculumSubjectPhaseOverviewQuery =
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  (sdk: Sdk) => async (args: { subject: string; phase: string }) => {
+  (sdk: Sdk) => async (args: { slug: string }) => {
+    const splitSlug = args.slug.split("-");
+    if (splitSlug.length < 2) {
+      throw new OakError({ code: "curriculum-api/not-found" });
+    }
+    const [subjectSlug, phaseSlug] = splitSlug;
```

Comment:
Some subjects can have multiple words, so this might need to be something like: 
```
const split = slug.lastIndexOf('-');
if (!split) {
  throw new OakError({ code: "curriculum-api/not-found" });
}
const subjectSlug = str.substring(0, split);
const phaseSlug = str.substring(split + 1);
```